README.md

```diff
@@ -32,6 +32,12 @@ Space Station 13 is a community developed, multiplayer round-based role playing
 
 [Mapping Guidelines](https://hackmd.io/@ZeWaka/goonmap)
 
+## FAQ (Frequently Asked Question)
```

Comment:
s

Reply:
You see, it was supposed to be a joke because that's literally the only question I've seen asked frequently.